web_src/less/_base.less

```diff
@@ -1382,6 +1382,14 @@ a.ui.label:hover {
   margin-right: 0;
 }
 
+.lines-blame-btn {
+  padding-left: 10px;
+  padding-right: 10px;
+  text-align: right !important;
+  background-color: #f5f5f5;
```

Comment:
This background color needs to overridden in `web_src/less/themes/them-arc-green.less` L464:
```
.blame .lines-num,
.blame .lines-blame-button {
```

Reply:
i think we should use css vars

Reply:
silverwind did the transition to them for colors ...

Reply:
In that case, the existing override for `.blame .lines-num` in `theme-arc-green.less` can probably dropped